src/main/java/uk/gov/justice/digital/hmpps/whereabouts/config/ResourceServerConfiguration.java

```diff
@@ -66,7 +67,7 @@ public Docket api() {
                 getVersion(), "", contactInfo(), "", "",
                 Collections.emptyList());
 
-        final var docket = new Docket(DocumentationType.SWAGGER_2)
+        final var docket = new Docket(DocumentationType.OAS_30)
```

Comment:
this moves the swagger ui to consume from the openAPI json resource rather than the swagger resource